packages/vx-axis/src/utils/radialTickLabelTransform.js

```diff
@@ -0,0 +1,28 @@
+const CENTER_ALIGN_THRESHOLD = 1;
+
+export default function radialTickLabelTransform(fromPoint, toPoint) {
+  let textAnchor = 'middle';
+  let verticalAnchor = 'middle';
+  const dx = toPoint.x - fromPoint.x;
+  const dy = toPoint.y - fromPoint.y;
+
+  // offset the label in the same direction as the tick orientation
+  const x = toPoint.x + dx / 2;
```

Comment:
this offset is somewhat arbitrary but gave decent results as you can see in the demo tile. we could possibly make it more configurable.